tests/rkt_socket_proxyd_test.go

```diff
@@ -34,6 +34,10 @@ import (
 )
 
 func TestSocketProxyd(t *testing.T) {
+	// Skip the test for now. See https://github.com/coreos/rkt/issues/1464 for
+	// details.
```

Comment:
Shouldn't you link to this too?
https://github.com/coreos/rkt/issues/2432#issuecomment-238858840